{"name": "test_login[case1]", "status": "failed", "statusDetails": {"message": "AssertionError: assert '请求访问：/system/user/getUserInfo，认证失败，无法访问系统资源' == None", "trace": "self = <test_login.TestLogin object at 0x1032a8e90>, case = {'data': None, 'expected': None, 'files': None, 'headers': None, ...}\n\n    @pytest.mark.parametrize(\"case\",data)\n    def test_login(self,case):\n        case_data = get_rest(case)\n        request_data = case_data\n    \n        res = requests.request(**request_data)\n>       assert res.json()[\"msg\"] == case[\"expected\"]\nE       AssertionError: assert '请求访问：/system/user/getUserInfo，认证失败，无法访问系统资源' == None\n\nscripts/test_login.py:16: AssertionError"}, "parameters": [{"name": "case", "value": "{'method': 'get', 'path': 'prod-api/system/user/getUserInfo', 'headers': None, 'params': None, 'data': None, 'json': None, 'files': None, 'expected': None}"}], "start": 1736850023152, "stop": 1736850023556, "uuid": "5e7d8deb-a540-449f-a791-d0cc6de54ba9", "historyId": "9be4185697347804d75919d3201612e9", "testCaseId": "1d80d88536c622141c7393804df49bc9", "fullName": "scripts.test_login.TestLogin#test_login", "labels": [{"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "MacBook-Pro.local"}, {"name": "thread", "value": "16056-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.test_login"}]}